core/Tests.fs

```diff
@@ -271,7 +271,7 @@ type Results = {passes:int; failures:int; errors:int}
 
 [<EntryPoint>]
 let main argv =
-  let norm (s: String) = s.ToLower().Replace('\\', '/')
+  let norm (s: String) = s.Replace('\\', '/')
```

Comment:
This fixes the tests trying to read differing cased files.

Before this change:
```sh
$ node .config/do.mjs test
Error: ENOENT: no such file or directory, open './docs/specs/readme.md'
```

The actual file name is `docs/specs/README.md`

Tested on Linux (Manjaro, arch-based)